### DISCUSSION TITLE
Shelly Plus 1 without Bluetooth.

### DISCUSSION THREAD

**Discussion by dippes** [arendst/Tasmota#21600] [Q&A]
According to the description, my Shelly Plus 1 is Bluetooth capable. Which Tasmota firmware must be installed? Will any functions be lost?

**Comment by kroon040** [comment]
Default build is not available, but here are tasmota-specials and there is a tasmota32solo1-bluetooth

https://github.com/Jason2866/Tasmota-specials/tree/firmware/firmware/tasmota32/other
